mergify_engine/tests/functional/base.py

```diff
@@ -198,11 +198,15 @@ def _match(cls, data, expected_data):
     def _get_events(self):
         # NOTE(sileht): we use a counter to make each call unique in cassettes
         self._counter += 1
-        return self._session.get(
+        r = self._session.get(
             f"https://gh.mergify.io/events-testing?counter={self._counter}",
             data=FAKE_DATA,
             headers={"X-Hub-Signature": "sha1=" + FAKE_HMAC},
-        ).json()
+        )
+        try:
+            return r.json()
+        except json.decoder.JSONDecodeError:
+            raise Exception(r.text)
```

Comment:
```suggestion
            raise RuntimeError("Unable to decode JSON", r.text)
```

Reply:
That was for debugging, the aioredis bug, I removed it.